### DISCUSSION TITLE
Extension Support Request: Roop

### DISCUSSION THREAD

**Discussion by ryanbrackney** [AbdBarho/stable-diffusion-webui-docker#517] [Ideas]
I'd like to request support for the new face swapping extension, Roop. https://github.com/s0md3v/sd-webui-roop to the AUTO image. 

It requires a module "insightface", which in turn requires GCC. I ran into this error trying to install insightface, and decided to stop before I broke it:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
open-clip-torch 2.7.0 requires protobuf==3.20.0, but you have protobuf 4.23.3 which is incompatible.
```

**Comment by AbdBarho** [comment]
you can try with the `startup.sh` script

https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Usage#extensions
